proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -62,15 +61,19 @@ service BeaconChain {
 
     // Server-side stream of attestations as they are received by
     // the beacon chain node.
+    // DEPRECATED: This endpoint is superseded by the /eth/v1/events Beacon API endpoint, there is an internal GRPC endpoint /internal/eth/v1/events
```

Comment:
We shouldn't encourage users to use `internal` endpoints because they might not work properly. They should use the REST interface.